backend/core/management/commands/remind_registrations.py

```diff
@@ -0,0 +1,112 @@
+created.
+"""
+
+from smtplib import SMTPException
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Count, Q
+from django.template.loader import render_to_string
+from django.utils import timezone, translation
+from django.utils.translation import gettext_lazy as _
+
+from core.models.user import User
+
+
+class Command(BaseCommand):
+    help = "Notify by email active users who haven't contributed since their"\
+           " account was created."
+
+    def get_subject(self):
+        return "🌻 " + _("Want to help responsible scientific research?")
+
+    def get_message(self, n_comparisons, ctx):
+        if n_comparisons > 0:
+            return render_to_string("core/no_contrib_email/body_signup_contrib.txt", ctx)
+        return render_to_string("core/no_contrib_email/body_no_contrib.txt", ctx)
+
+    def get_html_message(self, n_comparisons, ctx):
+        if n_comparisons > 0:
+            return render_to_string("core/no_contrib_email/body_signup_contrib.html", ctx)
+        return render_to_string("core/no_contrib_email/body_no_contrib.html", ctx)
+
+    def notify_users(self, users):
+        from_email = settings.REST_REGISTRATION["VERIFICATION_FROM_EMAIL"]
+
+        ctx = {
+            "ext_url_firefox": getattr(settings, "BROWSER_EXT_URL_FIREFOX", None),
+            "ext_url_chrome": getattr(settings, "BROWSER_EXT_URL_CHROME", None)
+        }
+
+        fails = []
+        successes = 0
+
+        for user in users:
+            if user.n_comp_after_signup > 0:
+                continue
+
+            try:
+                lang_code = user.settings["general"]["notifications__lang"]
+            except KeyError:
+                lang_code = "en"
+
+            translation.activate(lang_code)
+
+            try:
+                send_mail(
+                    subject=self.get_subject(),
+                    message=self.get_message(user.n_comp_signup, ctx),
+                    html_message=self.get_html_message(user.n_comp_signup, ctx),
+                    from_email=from_email,
+                    recipient_list=[user.email],
+                    fail_silently=False,
+                )
+            except SMTPException as ex:
+                fails.append((user.email, type(ex).__name__, ex))
+            else:
+                successes += 1
+
+        return successes, fails
+
+    def get_queryset(self, creation_date):
+        return User.objects.filter(
+            is_active=True,
+            date_joined__date=creation_date.date(),
+        ).annotate(
+            n_comp_signup=Count(
+                "comparisons",
+                filter=Q(comparisons__datetime_add__date=creation_date.date()),
+            ),
+            n_comp_after_signup=Count(
+                "comparisons",
+                filter=Q(comparisons__datetime_add__date__gt=creation_date.date()),
```

Comment:
Maybe it would be more robust to consider comparaisons made a few hours after `date_joined` here, instead of the arbitrary UTC date?

Reply:
Good idea.

How about 12 hours after the registration? It gives one morning to users that subscribed just before midnight to make comparisons. 24 hours look also good to me.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1884/commits/cf1cbaa0232d416540feb50ddd69d99758d5ba7e

I chose a period of 16 hours.